deployment/constructor_params/lynx/lynx-alpha-13-root-params.json

```diff
@@ -0,0 +1,24 @@
+{
+  "LynxStakingToken": {
+    "_totalSupplyOfTokens": 10000000000000000000000000
+  },
+  "TestnetThresholdStaking": {},
+  "ProxyAdmin": {},
+  "TACoApplication": {
+    "_token": "$LynxStakingToken",
+    "_tStaking": "$TestnetThresholdStaking",
+    "_minimumAuthorization": 40000000000000000000000,
+    "_minOperatorSeconds": 3600,
+    "_rewardDuration": 604800,
+    "_deauthorizationDuration": 5184000,
+    "_commitmentDurationOptions": [15724800, 31449600]
+  },
+  "TransparentUpgradeableProxy": {
+    "_logic": "$TACoApplication",
+    "admin_": "$ProxyAdmin",
+    "_data": "$bytes:0x"
```

Comment:
Still thinks that in the same way the parser needs to process `"$bytes:0x"` into `b""`, it would be simpler to try to process `""` into `b""`, since we already know the type of `_data`. In any case, not for this PR. We can definitely accept current version.